en/components/grids_and_lists.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Data Grids and Lists Components
+_description: Ignite UI for Angular provides the fastest Angular Data Grid, high-speed rendering, and Material-based components for a powerful Angular grid and features. 
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Grid, Angular Data Grid, Angular Grid Control, Angular Grid Component
+---
+
+<div class="feature feature--hero">
+  <div class="feature__details">
+    <h2 id="the-fastest-data-angular-data-grid">The Fastest Angular Data Grid</h2>
+    <p>Ignite UI for Angular’s <a href="/components/grid.html">Data Grid</a> is built for optimization, speed, high-performance, and smooth handling of large data sets. With an updated virtualization directive and powerful grid features, your data grid is ready for no-lag scrolling through millions of data points.</p>
```

Comment:
For links to other topics please use the following syntax `[Data Grid](grid.md)`.